secure_collaborative_computation_service/client_service/stub/chart/templates/deployment.yml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "service.fullname" . }}
+  labels:
+    app: {{ include "service.fullname" . }}
+spec:
+  replicas: {{ .Values.deployment.replicas }}
+  selector:
+    matchLabels:
+      app: {{ include "service.fullname" . }}
+  template:
+    metadata:
+      labels:
+        app: {{ include "service.fullname" . }}
+    spec:
+      containers:
+        - name: {{ include "service.fullname" . }}
+          image: {{ .Values.deployment.image }}
+          env:
+            - name: SERVICE_PORT
```

Comment:
it is better to add these environment variables to config maps than writing it here directly

Reply:
Thank you very much for the suggestion. In this particular case I think a config map is not the right tool to use, since the SERVICE_PORT environment variable has to match the targetPort in the kubernetes service. I added a helm config option to be able to change this environment variable if needed.